schema/project-level/project-schema.json

```diff
@@ -2370,6 +2390,200 @@
         }
       },
       "minProperties": 1
+    },
+    "Finance": {
+      "title": "Financing arrangement",
+      "description": "An arrangement between two parties for raising money.",
+      "type": "object",
+      "required": [
+        "id"
+      ],
+      "properties": {
+        "id": {
+          "title": "ID",
+          "description": "An identifier for this financing arrangement.",
+          "type": "string",
+          "minLength": 1
+        },
+        "title": {
+          "title": "Title",
+          "description": "A short title for this financing arrangement.",
+          "type": "string",
+          "minLength": 1
+        },
+        "description": {
+          "title": "Description",
+          "description": "A short description of this financing arrangement. This may include details of service fees, commitment fees, lender rights, step in arrangements, exchange rate guarantees, and for syndicated loans, details of syndication and any members of the syndicate.",
```

Comment:
```suggestion
          "description": "A short description of the financing arrangement. This may include details of service fees, commitment fees, lender rights, step in arrangements, exchange rate guarantees, and for syndicated loans, details of syndication and any members of the syndicate.",
```